tests/test_cooling_effect.py

```diff
@@ -22,7 +22,7 @@ def test_cooling_effect(get_cooling_effect_url, retrieve_data, is_equal):
 
         # To determine whether the result is as expected, use np.allclose for arrays and np.isclose for single
         try:
-            is_equal(result, expected_output, tolerance.get("cooling_effect"))
+            is_equal(result, expected_output, tolerance.get("cooling_effect", 1e-6))
```

Comment:
we discussed this yesterday and we should remove the string cooling effect from here. Instead, we should be looping through the keys in tolerance and from there we should be getting the information about the tolerance. You have implemented this in some of the test functions